bin/riemann-docker-health

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env ruby
+
+# Reports current CPU, disk, load average, and memory use to riemann.
+
+require File.expand_path('../../lib/riemann/tools', __FILE__)
+require 'docker'
+
+class Riemann::Tools::DockerHealth
+  include Riemann::Tools
+  include Docker
+
+  opt :docker_host, "Docker Container Host", :default => "localhost"
+  opt :cpu_warning, "CPU warning threshold (fraction of total jiffies)", :default => 0.9
+  opt :cpu_critical, "CPU critical threshold (fraction of total jiffies)", :default => 0.95
+  opt :disk_warning, "Disk warning threshold (fraction of space used)", :default => 0.9
+  opt :disk_critical, "Disk critical threshold (fraction of space used)", :default => 0.95
+  opt :memory_warning, "Memory warning threshold (fraction of RAM)", :default => 0.85
+  opt :memory_critical, "Memory critical threshold (fraction of RAM)", :default => 0.95
+  opt :checks, "A list of checks to run.", :type => :strings, :default => ['cpu', 'load', 'memory', 'disk']
+
+  def get_containers
+    Docker::Container.all
+  end
+
+  def get_container_name(container)
+    container.json["Name"][1..-1]
+  end
+
+  def initialize
+
+    @cpu_coefficient = 1000 * 1000 * 1000
+
+    @limits = {
+      :cpu => {:critical => opts[:cpu_critical], :warning => opts[:cpu_warning]},
+      :disk => {:critical => opts[:disk_critical], :warning => opts[:disk_warning]},
+      :memory => {:critical => opts[:memory_critical], :warning => opts[:memory_warning]}
+    }
+
+    @last_cpu_reads = Hash.new
+    @cpu = method :linux_cpu
```

Comment:
I should remove the `@cpu`, `@disk` and `@memory` variables, they are not needed because there is only one way to figure out them.